docker/run.sh

```diff
@@ -12,7 +12,7 @@ else
 fi
 
 if [ -n "${DD_AGENT_HOST+x}" ];then
-  DD_ARGS="-Ddd.agent.host=$DD_AGENT_HOST -Ddd.agent.port=$DD_TRACE_AGENT_PORT -Ddd.profiling.enabled=$DD_PROFILING_ENABLED -Ddd.logs.injection=true -Ddd.trace.sample.rate=1 -Ddd.service=wallet-daemon"
+  DD_ARGS="-Ddd.agent.host=$DD_AGENT_HOST -Ddd.agent.port=${DD_TRACE_AGENT_PORT:-8126} -Ddd.profiling.enabled=${DD_PROFILING_ENABLED:-true} -Ddd.logs.injection=true -Ddd.trace.sample.rate=1 -Ddd.service=wallet-daemon"
```

Comment:
One could argue that default values in the middle of a script line might be hard to find.
Maybe instead we could define default values just above?
i.e:
```suggestion
  DD_PROFILING_ENABLED=${DD_PROFILING_ENABLED:-true}
  DD_TRACE_AGENT_PORT=${DD_TRACE_AGENT_PORT:-8126}
  DD_ARGS="-Ddd.agent.host=$DD_AGENT_HOST -Ddd.agent.port=$DD_TRACE_AGENT_PORT -Ddd.profiling.enabled=$DD_PROFILING_ENABLED -Ddd.logs.injection=true -Ddd.trace.sample.rate=1 -Ddd.service=wallet-daemon"
```